usrv.c

```diff
@@ -47,12 +47,12 @@ static void usrv_msgthr(void *arg)
 
 		switch (oid.id) {
 			case USRV_ID_LOG:
-				log_msgHandler(&msg, oid, rid);
+				log_msgHandler(&msg, oid, (unsigned long)rid);
```

Comment:
IMHO we could update `log` to use `msg_rid_t` type (this function already takes `msg_t` as a param, so it "knows" about messages interface, no need to hide it). This should reduce the amount of casts inside `log.c` (i'm not flagging individual changes in `log.c` to be made